src/main/java/com/openlattice/datastore/search/controllers/SearchController.java

```diff
@@ -119,7 +119,7 @@ public SearchResult getEntitySets(
                         Optional.empty(),
                         Optional.empty(),
                         start,
-                        maxHits );
+                        Math.min( maxHits, SearchApi.MAX_SEARCH_RESULTS ) );
```

Comment:
This is max results per page, or max results total?

Reply:
max that you can request at one time